website/content/docs/connect/gateways/index.mdx

```diff
@@ -10,8 +10,8 @@ description: >-
 This topic provides an overview of the gateway features shipped with Consul. Gateways provide connectivity into, out of, and between Consul service meshes. You can configure the following types of gateways:
 
 - [Mesh gateways](#mesh-gateways) enable service-to-service traffic between Consul datacenters or between Consul admin partitions. They also enable datacenters to be federated across wide area networks.
-- [Ingress gateways](#ingress-gateways) enable services to accept traffic from outside the Consul service mesh.
-- [Terminating gateways](#terminating-gateways) enable you to route traffic from services in the Consul service mesh to external services.
+- [Ingress gateways](#ingress-gateways) enable services to accept traffic from outside the Consul service mesh that originates from your organizational network.
```

Comment:
**Question for reviewer**: Should I re-use the same wording that I use later (in the ingress gateway section)?
> Ingress gateways enable connectivity within your organizational network from services outside the Consul service mesh to services within the mesh.

I think that wording might be clearer than what's currently on this line...

Reply:
@karl-cardenas-coding : in that case, I'll change this line (and the terminating gateway line) to use the "within your organizational network ____________________________" structure before merging. Thanks!